packages/fuels/src/cli/commands/dev/autoStartFuelCore.ts

```diff
@@ -47,7 +46,7 @@ export const autoStartFuelCore = async (config: FuelsConfig) => {
       loggingEnabled: loggingConfig.isLoggingEnabled,
       debugEnabled: loggingConfig.isDebugEnabled,
       basePath: config.basePath,
-      useSystemFuelCore: !config.useBuiltinFuelCore,
+      useSystemFuelCore: true,
```

Comment:
I think this goes away; we won't have the concept of `system` vs. `built-in` anymore.